sdk/netapp/Azure.ResourceManager.NetApp/tests/ResourceTests/ANFBackupPolicyTests.cs

```diff
@@ -0,0 +1,204 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Azure.Core.TestFramework;
+using Azure.ResourceManager.NetApp.Models;
+using Azure.ResourceManager.NetApp.Tests.Helpers;
+using FluentAssertions;
+using NUnit.Framework;
+
+namespace Azure.ResourceManager.NetApp.Tests
+{
+    public class ANFBackupPolicyTests : NetAppTestBase
+    {
+        internal NetAppBackupPolicyCollection _backupPolicyCollection { get => _netAppAccount.GetNetAppBackupPolicies(); }
+
+        internal readonly NetAppBackupPolicyData _backupPolicy = new NetAppBackupPolicyData(DefaultLocation)
+        {
+            IsEnabled = true,
+            DailyBackupsToKeep = 4,
+            WeeklyBackupsToKeep = 3,
+            MonthlyBackupsToKeep = 2
+        };
+
+        public ANFBackupPolicyTests(bool isAsync) : base(isAsync)
+        {
+        }
+
+        [SetUp]
+        public async Task SetUp()
+        {
+            _resourceGroup = await CreateResourceGroupAsync();
+            string accountName = await CreateValidAccountNameAsync(_accountNamePrefix, _resourceGroup, DefaultLocation);
+            NetAppAccountCollection netAppAccountCollection = _resourceGroup.GetNetAppAccounts();
+            _netAppAccount = (await netAppAccountCollection.CreateOrUpdateAsync(WaitUntil.Completed, accountName, GetDefaultNetAppAccountParameters())).Value;
+        }
+
+        [TearDown]
+        public async Task ClearBackupPolicies()
+        {
+            //remove all backupPolicies under current netAppAccound and remove netAppAccount
+            if (_resourceGroup != null)
+            {
+                List<NetAppBackupPolicyResource> backupPolicyList = await _backupPolicyCollection.GetAllAsync().ToEnumerableAsync();
+                //remove backupPolicies
+                foreach (NetAppBackupPolicyResource backupPolicy in backupPolicyList)
+                {
+                    await backupPolicy.DeleteAsync(WaitUntil.Completed);
+                }
+                //remove account
+                if (Mode != RecordedTestMode.Playback)
+                {
+                    await Task.Delay(40000);
+                }
+                await _netAppAccount.DeleteAsync(WaitUntil.Completed);
+            }
+            _resourceGroup = null;
+        }
+
+        [Test]
+        [RecordedTest]
```

Comment:
I'm not sure which one, but if there has race condition issue, can try `NonParallelizable`

Reply:
updated, seems to work now